packages/deployment/task-reaper.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+set -e
+
+echo "$(date) Starting task-reaper.sh"
+
+INSTANCE_ID=$(curl http://169.254.169.254/latest/meta-data/instance-id)
+
+CLUSTER=$(grep 'ECS_CLUSTER=' /etc/ecs/ecs.config | cut -d '=' -f 2)
+
+CONTAINER_INSTANCE_ARN=$(aws ecs list-container-instances --cluster "$CLUSTER" --filter "ec2InstanceId == $INSTANCE_ID" | jq -r '.containerInstanceArns[0]')
+CONTAINER_INSTANCE_STATUS=$(aws ecs describe-container-instances --cluster "$CLUSTER" --container-instances "$CONTAINER_INSTANCE_ARN" | jq -r .containerInstances[0].status)
```

Comment:
I think this is fine because we'd only ever expect to get one container from a `--container-instances "$CONTAINER_INSTANCE_ARN"`.